wasp/wasp.py

```diff
@@ -595,4 +596,41 @@ def theme(self, theme_part: str) -> int:
         idx = theme_parts.index(theme_part) * 2
         return (self._theme[idx] << 8) | self._theme[idx+1]
 
+    def store_settings(self, name, value):
+        """Stores any variable as string in a text file in a 'settings'
+        folder, making it persistent across reboots. For example: used
+        to store alarms or stopwatch start time.
+        If value is a list, the items will be separated by a ";". Hence, ";" is
+        not supposed to appear in 'value'.
+        """
+        if "settings" not in os.listdir():
+            os.mkdir("settings")
+        try:
+            with open("settings/" + name, "w") as f:
+                if isinstance(value, list):
+                    f.write(";".join([str(x) for x in value]))
+                else:
+                    f.write(str(value))
+            assert name in os.listdir("settings/")
+            return True
+        except Exception as err:
+            self.notify(watch.rtc.get_uptime_ms(), {
+                "src": "Persistent settings",
+                "title": "Storing failed",
+                "body": "Error when storing '{}': '{}'".format(name, err)})
+
+    def get_settings(self, name, delete=False):
```

Comment:
Adding default = None might make it better for some callers (and then return default rather than None in the error paths).

Reply:
I actually don't understand what you mean by "default" or how people could use it.